From 36522a7c0e0a3b2bf4419cefb6cc20b40c66d975 Mon Sep 17 00:00:00 2001
From: David Runge <dave@sleepmap.de>
Date: Wed, 15 Jul 2026 09:29:58 +0200
Subject: [PATCH] Prevent clash of _FORTIFY_SOURCE definition with distribution
 flags

Currently, we define `-D_FORTIFY_SOURCE=3` in our distribution flags.
Without this modification, the build would fail.

Signed-off-by: David Runge <dvzrv@archlinux.org>
---
 cmake/SecurityFlags.cmake | 1 -
 1 file changed, 1 deletion(-)

diff --git a/cmake/SecurityFlags.cmake b/cmake/SecurityFlags.cmake
index 01505e0..747326a 100644
--- a/cmake/SecurityFlags.cmake
+++ b/cmake/SecurityFlags.cmake
@@ -14,7 +14,6 @@ if (CMAKE_C_COMPILER_ID STREQUAL "Clang" OR
     if (NOT FUZZING)
         add_compile_options (-Wall -Wextra -Werror)
         add_compile_options(-O2)
-        add_definitions (-D_FORTIFY_SOURCE=2)
     endif ()
 
     check_c_compiler_flag("-fstack-protector-all" HAVE_STACK_PROTECTOR_ALL)
